crates/torii/src/engine.rs

```diff
@@ -51,10 +53,11 @@ impl<'a, S: State + Executable, T: JsonRpcTransport + Sync + Send> Engine<'a, S,
         storage: &'a S,
         provider: &'a JsonRpcClient<T>,
         processors: Processors<S, T>,
+        world_address: FieldElement,
```

Comment:
What do you think of moving this and `start_block` to the `EngineConfig`?

Reply:
Yep, make sense